src/main/resources/static/javascript/mainApp/general.controller.js

```diff
@@ -265,6 +267,23 @@
         $scope.hoverCopy = () => {
             $("[data-content='copy']").popover();
         };
+
+        /**
+         * Updates the active user profile, clears cache by reloading the page with a cache-busting URL.
+         * @param profile - the type of profile to switch to ('MEMBER', 'OWNER', 'ADMIN')
+         */
+        $scope.updateActiveUserProfile = (profile) => {
+            groupingsService.updateOotbActiveProfile(profile, () => {
+                $scope.$apply(() => {
+
+                });
+            }, (res) => {
+                $scope.$apply(() => {
+                    $scope.resStatus = res.status;
+                });
+            });
```

Comment:
Sorry for another comment on the same place, just realized this can simply be:
```
groupingsService.updateOotbActiveProfile(profile, 
        () => $scope.$apply(),
        (res) => {
            $scope.resStatus = res.status;
         });
```
The onSuccess callback does not need anything inside the $scope.$apply(). The onError function does not to call $scope.$apply since it failed to update the current OOTB user.